src/components/BookContainer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  Routes,
+  Route,
+} from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import BookList from './BookList';
+import BookInput from './BookInput';
+import Category from './Category';
+
+function BookContainer() {
+  const books = useSelector((state) => state.books);
```

Comment:
List of books displays books from Redux store (by using `useSelector` function) 💪😬